downloads/windows.md

```diff
@@ -43,7 +43,7 @@ Running Iridium portable version works out of the box. As a recommendation we re
 If you are looking for previous versions, please check out the [Builds Archive](https://downloads.iridiumbrowser.de/windows/ "go to Builds Archive"){:target="_blank"}<br/>
 \\
 system requirements   
-&#8226; Windows 7+   
+&#8226; Windows 7 or later   
```

Comment:
You can't have one place say "macOS 10.9+" and another "7 or later". You need to decide for one style.

Reply:
agreed & fixed!